src/app/Fake.Windows.Chocolatey/Chocolatey.fs

```diff
@@ -822,7 +828,31 @@ module Fake.Windows.Choco
                 |> appendWithoutQuotesIfNotNull parameters.Source "--source "
                 |> appendWithoutQuotesIfNotNull parameters.ApiKey "--apikey "
                 |> appendIfTrueWithoutQuotes parameters.NonInteractive "-y"
-                |> appendWithoutQuotesIfNotNull parameters.AdditionalArgs parameters.AdditionalArgs
+                |> appendIfTrueWithoutQuotes parameters.Force "--force"
+                |> appendIfTrueWithoutQuotes (parameters.AdditionalArgs |> String.isNotNullOrEmpty) parameters.AdditionalArgs
                 |> toText
 
         callChoco parameters.ToolPath args parameters.Timeout
+                
+    /// Call custom choco command
+    /// ## Parameters
+    ///  - `args` - string that will be appendedn to choco.exe call
+    ///  - `timeout` - parrent process maximum completion time
+    /// ## Sample usage
+    ///
+    ///     Target "ChocoPush" (fun _ ->
+    ///
+    ///          let newSpecFile = ...
+    ///          let args = 
+    ///                 new StringBuilder()
+    ///                 |> append "pack"
+    ///                 |> append newSpecFile
+    ///                 |> append "-y"
+    ///                 |> toText
+    ///        
+    ///         args |> Choco.CallChoco TimeSpan.FromMinutes 1.
+    ///     )
+    let CallChoco args timeout =
```

Comment:
Maybe "Exec" so that it reads `Choco.Exec` in the code? (currently it will read `Choco.CallChoco`)